src/data/optimizely.js

```diff
@@ -1,11 +1,34 @@
-import {
-  createInstance,
-} from '@optimizely/react-sdk';
+import { createInstance } from '@optimizely/react-sdk';
+import { ensureConfig, getConfig } from '@edx/frontend-platform';
 
-const OPTIMIZELY_SDK_KEY = process.env.OPTIMIZELY_FULL_STACK_SDK_KEY;
+ensureConfig([
+  'OPTIMIZELY_FULL_STACK_SDK_KEY',
+], 'Frontend Lib Learning Assistant Optimizely module');
 
-const optimizely = createInstance({
-  sdkKey: OPTIMIZELY_SDK_KEY,
-});
+/**
+ * Initializing the Optimizely instance at the top-level will not work, becaused it may be initialized before
```

Comment:
I have myself to blame for this type in the original implementation, but we may as well fix it here when we're adding it!

```suggestion
 * Initializing the Optimizely instance at the top-level will not work, because it may be initialized before
```

Reply:
Thanks. My bad. Copy-paste bad habits.